### DISCUSSION TITLE
DB/db11/

### DISCUSSION THREAD

**Discussion by giscus** [HobbitQia/notebook#53] [General]
# DB/db11/

HobbitQia的笔记本

https://note.hobbitqia.cc/DB/db11/

**Comment by 0AyanamiRei** [comment]
非常好的配图，数据库概念原作上的图和内容隔得太远，非常难对应上 qwq

**Comment by Hongjin-Tian** [comment]
学长，这几天数据库笔记的图片看不了了，无论是换设备还是换浏览器都无法解决
